### DISCUSSION TITLE
Open HoverCard with click too

### DISCUSSION THREAD

**Discussion by ViniDevBR** [radix-ui/primitives#1884] [Help] [answered]
I´m using hoverCard, and that is perfect, but in mobile we dont have HOVER. How open HoverCard with click too?

**Comment by ViniDevBR** [comment]
Like a boolean option, onClickOpen

**Reply by benoitgrelard** [reply, marked as answer]
Hey @ViniDevBR, that wouldn't make sense because the trigger is a link, so clicking on it would navigate away. `HoverCard` should be treated purely as a progressive enhancement for showing info that is already available on behind that link. Think the user cards on twitter or github, when you click on them on touch screens, you'll see it simply navigates to that user's page.

**Reply by vnugent** [reply]
I came here to ask the same question.  Perhaps is there a way to open hover card on [long press](https://m3.material.io/foundations/interaction/gestures)?  Another word, on long press, behave like a popover.  Our app contains a lot of hierarchical pages.  The card enables our power users to jump between sibling pages.  I'm hoping if there's a simpler/elegant solution besides "if mobile then use popover, else use hover card".
